worlds/stardew_valley/regions.py

```diff
@@ -353,8 +354,81 @@ def inverted(self):
+                for entrance in mod_data.connections:
+                    if mod_data.flags[entrance] == "NON_RANDOMIZED":
+                        loaded_mod_data.append(ConnectionData(entrance, mod_data.connections[entrance],
+                                                              flag=RandomizationFlag.NOT_RANDOMIZED))
+                    if mod_data.flags[entrance] == "NON_PROGRESSION":
+                        loaded_mod_data.append(ConnectionData(entrance, mod_data.connections[entrance],
+                                                              flag=RandomizationFlag.NON_PROGRESSION))
+                    if mod_data.flags[entrance] == "PELICAN_TOWN":
+                        loaded_mod_data.append(ConnectionData(entrance, mod_data.connections[entrance],
+                                                              flag=RandomizationFlag.PELICAN_TOWN))
+                    if mod_data.flags[entrance] == "BUILDINGS":
+                        loaded_mod_data.append(ConnectionData(entrance, mod_data.connections[entrance],
+                                                              flag=RandomizationFlag.BUILDINGS))
+                    if mod_data.flags[entrance] == "EVERYTHING":
+                        loaded_mod_data.append(ConnectionData(entrance, mod_data.connections[entrance],
+                                                              flag=RandomizationFlag.EVERYTHING))
+            if (mod_data.mod_name == mod) & (data_type == "Region Remover"):
+                for name in mod_data.region_remover:
+                    loaded_mod_data.append(RegionRemoverData(name, mod_data.region_remover[name]))
+            if (mod_data.mod_name == mod) & (data_type == "Connection Remover"):
+                for name in mod_data.connection_remover:
+                    loaded_mod_data.append(ConnectionRemoverData(name, mod_data.region_remover[name]))
+    return loaded_mod_data
+
+
+def region_extend(mod: str, existing_regions: List[Region], new_regions: List[Region], world_options: StardewOptions):
+    if mod not in world_options[options.Mods]:
+        return
+    to_remove = loaded_mod_region_data(mod, "Region Remover", world_options)
+    for regions_to_remove in to_remove:
+        for region in existing_regions:
+            if regions_to_remove.name == region.name:
+                for exits_to_remove in regions_to_remove.exits:
+                    region.exits.remove(exits_to_remove)
+    for new_region in new_regions:
+        existing_region = next(
+            (region for region in existing_regions if region.name == new_region.name), None)
+        if existing_region:
+            if all(exits in existing_region.exits for exits in new_region.exits):
+                continue
+            existing_region.exits.extend(new_region.exits)
+            continue
+
+        existing_regions.append(new_region)
+
+
+def connection_patcher(mod: str, old_connections: List[ConnectionData], world_options: StardewOptions):
```

Comment:
Same goes for this method. I expect this to go away or pull a Phoenix Man and shrink to completely stop being a problem